app/controllers/twilio_controller.rb

```diff
@@ -0,0 +1,18 @@
+require 'twilio-ruby'
+
+class TwilioController < ApplicationController
+  include Webhookable
+
+  after_filter :set_header
+
+  skip_before_action :verify_authenticity_token
+
+  def voice
+  	response = Twilio::TwiML::Response.new do |r|
+  	  r.Say 'Hey there. Congrats on integrating Twilio into your Rails 4 app.', :voice => 'alice'
+         r.Play 'http://linode.rabasa.com/cantina.mp3'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.